### PR TITLE
Implement Portfolio and Transactions tab structure

### PR DESCRIPTION
**Pull request by dmkirshon**
## Update:
- Wallet page to include a Portfolio tab and a Transactions tab for the wallet enhancements and earning claiming

https://projectnewm.atlassian.net/browse/STUD-93

### CODE REVIEW

File: apps/studio/src/pages/home/wallet/Wallet.tsx [hunk 1]
```diff
@@ -1,26 +1,69 @@
...
     <Container maxWidth={ false }>
-      <Box ml={ [null, null, 3] }>
+      <Box mx={ [null, null, 1.5] }>
         <Box
           sx={ {
             alignItems: "center",
             display: "flex",
             justifyContent: "space-between",
-            mb: 5,
+            marginBottom: 5,
```
Comment (escobarjonatan) [inline apps/studio/src/pages/home/wallet/Wallet.tsx]: y u no like mb?

File: apps/studio/src/pages/home/wallet/Wallet.tsx [hunk 2]
```diff
@@ -41,10 +83,49 @@ const Wallet: FunctionComponent = () => {
           ) }
         </Box>
 
-        <UnclaimedRoyalties unclaimedRoyalties={ 0 } />
+        { !!unclaimedRoyalties && (
```
Comment (escobarjonatan) [inline apps/studio/src/pages/home/wallet/Wallet.tsx:86]: do we want to show something when the opposite is true?

**Review by escobarjonatan — APPROVED** [review]

File: apps/studio/src/pages/home/wallet/Wallet.tsx (hunk 1 above)
Comment (dmkirshon) [reply apps/studio/src/pages/home/wallet/Wallet.tsx]: I was playing around with the other padding and margins, just didn't catch it to revert to mb, will fix to not inflict more changes

File: apps/studio/src/pages/home/wallet/Wallet.tsx (hunk 2 above)
Comment (dmkirshon) [reply apps/studio/src/pages/home/wallet/Wallet.tsx:86]: From what I can tell from the Figma designs, not right now. Will have to look into the other tickets, as this one was just for the tab structure. I will throw the question to design and product but I went for basic appearance for this ticket.

**Pull request merged**